specification/datadog/resource-manager/Microsoft.Datadog/preview/2020-02-01-preview/datadog.json

```diff
@@ -1022,6 +1111,87 @@
     }
   },
   "definitions": {
+    "DatadogAgreementProperties": {
+      "description": "Terms properties.",
+      "type": "object",
+      "properties": {
+        "publisher": {
+          "description": "Publisher identifier string.",
+          "type": "string"
+        },
+        "product": {
+          "description": "Product identifier string.",
+          "type": "string"
+        },
+        "plan": {
+          "description": "Plan identifier string.",
+          "type": "string"
+        },
+        "licenseTextLink": {
+          "description": "Link to HTML with Microsoft and Publisher terms.",
+          "type": "string"
+        },
+        "privacyPolicyLink": {
+          "description": "Link to the privacy policy of the publisher.",
+          "type": "string"
+        },
+        "retrieveDatetime": {
+          "format": "date-time",
+          "description": "Date and time in UTC of when the terms were accepted. This is empty if Accepted is false.",
+          "type": "string"
+        },
+        "signature": {
```

Comment:
this and properties above should be marked read-only as you do not accept them on the put, right?